docs/source/howto/load-singlesshagent.sh

```diff
@@ -0,0 +1,68 @@
+# Make sure you source this script rather than executing it
```

Comment:
Is this a new feature? Then we should probably not introduce it as part of the docs-revamp.

Reply:
It is not a new feature, but it is definitely new documentation. But anyway we are creating new docs for other how-to pages, and this is very timely considering the recent cyberattack (actually Nicola Spallanzani who contributed most of the text contacted me about this, worried that it was not possible to run with AiiDA) so I feel it is important to put it in the docs ASAP, as many more users will have the exact same question (already some of us developers were worried that AiiDA does not support passphrase-protected keys).

Also, I've been using the script for the past 3 years, and Nicola Spallanzani tried it and confirmed that it works. So I'm quite confident that it is working, or in the worst case can serve as a starting point for users to write their own adaptation.

Anyway, if this is controversial and delaying the merge of the PR, I'll remove it and make an independent PR.

Reply:
Let's leave it in to avoid creating work.